### PR TITLE
Activities: re-organize some classes

### PR DESCRIPTION
**Pull request by Betree**
@kewitz I re-organized some activities in different classes, let me know if that makes sense to you

### CODE REVIEW

File: server/constants/activities.ts [hunk 1]
```diff
@@ -125,14 +125,21 @@ export const ActivitiesPerClass: Record<ActivityClasses, ActivityTypes[]> = {
     ActivityTypes.COLLECTIVE_APPLY,
     ActivityTypes.COLLECTIVE_APPROVED,
     ActivityTypes.COLLECTIVE_CORE_MEMBER_INVITED,
+    ActivityTypes.COLLECTIVE_CORE_MEMBER_INVITATION_DECLINED,
+    ActivityTypes.COLLECTIVE_CORE_MEMBER_ADDED,
+    ActivityTypes.COLLECTIVE_CORE_MEMBER_EDITED,
+    ActivityTypes.COLLECTIVE_CORE_MEMBER_REMOVED,
```
Comment (Betree) [inline server/constants/activities.ts:131]: Added all core member activities to the "COLLECTIVE" class

File: server/constants/activities.ts [hunk 2]
```diff
@@ -175,14 +183,8 @@ export const ActivitiesPerClass: Record<ActivityClasses, ActivityTypes[]> = {
     ActivityTypes.COLLECTIVE_CONVERSATION_CREATED,
     ActivityTypes.COLLECTIVE_UPDATE_PUBLISHED,
     ActivityTypes.COLLECTIVE_UPDATE_CREATED,
-    ActivityTypes.ACTIVATED_COLLECTIVE_AS_HOST,
-    ActivityTypes.ACTIVATED_COLLECTIVE_AS_INDEPENDENT,
```
Comment (Betree) [inline server/constants/activities.ts:179]: Didn't make sense to me to have them under "Updates". Moved to `COLLECTIVE`

File: server/constants/activities.ts [hunk 3]
```diff
@@ -175,14 +183,8 @@ export const ActivitiesPerClass: Record<ActivityClasses, ActivityTypes[]> = {
...
     ActivityTypes.CONVERSATION_COMMENT_CREATED,
     ActivityTypes.UPDATE_COMMENT_CREATED,
-    ActivityTypes.ORGANIZATION_COLLECTIVE_CREATED,
-    ActivityTypes.COLLECTIVE_MEMBER_INVITED,
-    ActivityTypes.USER_CARD_INVITED,
-    ActivityTypes.USER_CARD_CLAIMED,
```
Comment (Betree) [inline server/constants/activities.ts:185]: These ones are for gift cards, I haven't moved there anywhere but it sounds like things we don't want people to unsubscribe from

File: server/constants/activities.ts [hunk 4]
```diff
@@ -175,14 +183,8 @@ export const ActivitiesPerClass: Record<ActivityClasses, ActivityTypes[]> = {
     ActivityTypes.COLLECTIVE_CONVERSATION_CREATED,
     ActivityTypes.COLLECTIVE_UPDATE_PUBLISHED,
     ActivityTypes.COLLECTIVE_UPDATE_CREATED,
-    ActivityTypes.ACTIVATED_COLLECTIVE_AS_HOST,
-    ActivityTypes.ACTIVATED_COLLECTIVE_AS_INDEPENDENT,
     ActivityTypes.CONVERSATION_COMMENT_CREATED,
     ActivityTypes.UPDATE_COMMENT_CREATED,
-    ActivityTypes.ORGANIZATION_COLLECTIVE_CREATED,
-    ActivityTypes.COLLECTIVE_MEMBER_INVITED,
```
Comment (Betree) [inline server/constants/activities.ts:183]: Sounded better to me to have these under `COLLECTIVE` than `UPDATE`

**Comment by kewitz** [comment]
I think it makes sense overall.
We need to check with Memo and Ronen, who had previously compiled a document classifying those activities.
Also, it may be interesting to check in the front end how these classes are described.

**Comment by iamronen** [comment]
seems like sensible cleanup to me :)

**Review by kewitz — APPROVED** [review]

**Pull request merged**